webapp/src/components/App/Singlestudy/explore/TableMode/index.tsx

```diff
@@ -17,6 +16,7 @@ import CreateTemplateTableDialog from "./dialogs/CreateTemplateTableDialog";
 import UpdateTemplateTableDialog from "./dialogs/UpdateTemplateTableDialog";
 import ConfirmationDialog from "../../../../common/dialogs/ConfirmationDialog";
 import TableMode from "../../../../common/TableMode";
+import SplitView from "../../../../common/SplitView";
 
 function TableModeList() {
```

Comment:
Keep the folder name `TableModeList`, because in the side of the code is a list of Table Mode